src/subscriber.go

```diff
@@ -76,9 +81,12 @@ func (s *Subscriber) Subscribe(publisher *Publisher) {
 	s.mutex.Lock()
 	s.Track = track
 	s.sender = sender
+	s.ssrc = uint32(sender.GetParameters().Encodings[0].SSRC)
```

Comment:
Thing is, we don't really want to store it if it's already part of the structure.

I.e. imagine you have something like this:
```
struct A {
    prop1: u32,
    prop2: u32,
}

struct Aggregator {
    a A,
}
```

Imagine you want to have access to the `prop2`.

Currently, the access to `prop2` is acquired like this:
```
struct Aggregator {
    a A,
    prop2: u32,
}
```

where `prop2` is equal to `a.prop2`.

The problem with duplicating this information is that you must programmatically ensure that `prop2` is essentially **a copy** of `a.prop2`, meaning that both must be kept in sync (any change to `prop2` must be reflected in `a.prop2`, any change of `a` must also update `prop2`. This complicates the life of a developer to keep them in sync 🙂

So instead, it's better to just access the `prop2` directly through the `a`. Or, in case this access requires more code, we create a **function** (basically helper/getter) that accesses the `a`'s field.

The only single reason why one may want to have **a copy** of a variable that is already part of the other structure that one has access to is for performance reasons (e.g. if acquiring the value involves some relatively expensive computation), but when it's the case the coupled values are normally extracted into a separate structure/module, the internal implementation of which ensures synced changed of both values (i.e. that both values are always identical) and provides faster access to the internal via e.g. some getters.

So generally speaking, if you can access property `X` of `Y` **while having access** (!) to `Y`, it's better to access it directly via `Y` instead of creating a copy of this property.

Does it make any sense? 🙂